src/Examples/Program.cs

```diff
@@ -141,6 +141,7 @@ internal static async Task _RunAndTrack(IZRunGame gameProcess)
 
         public static async void Mulitplayer(ZGame game)
         {
+            string strainthline = "_______________________________________________________________________________________";
```

Comment:
@nintend01337 Please comment on the code. This is an important part of this project. Also, try to follow the rules of a specific code style.